### PR TITLE
upgrade to gradle 6.5.1 & spring boot 2.3.1

### PR DESCRIPTION
**Pull request by bmscomp**
externilize versions on gradle.properties files 

upgrade gradle & spring boot version

update images resources for README.md file

### CODE REVIEW

File: initial/application/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.2.0.RELEASE</version>
+		<version>2.1.3.RELEASE</version>
```
Comment (Buzzardo) [inline initial/application/pom.xml]: This should be 2.3.1.

**Review by Buzzardo — CHANGES_REQUESTED** [review]
Please change that 2.1.3 to 2.3.1. Looks good otherwise.

File: initial/application/pom.xml (hunk 1 above)
Comment (bmscomp) [reply initial/application/pom.xml]: Thank you @Buzzardo , well seen , I fixed it with a new push

**Review by Buzzardo — APPROVED** [review]

**Comment by Buzzardo** [comment]
Thanks. :)

**Pull request merged**